module/zfs/spa.c

```diff
@@ -6593,6 +6619,17 @@ spa_vdev_attach(spa_t *spa, uint64_t guid, nvlist_t *nvroot, int replacing)
 	if (oldvd->vdev_top->vdev_islog && newvd->vdev_isspare)
 		return (spa_vdev_exit(spa, newrootvd, txg, ENOTSUP));
 
+	if (rebuild) {
+		/*
+		 * For rebuilds, the only allowable parents are a mirror,
+		 * the root vdev, or a dRAID with fixed width stripes.
```

Comment:
Since dRAID is not yet present, perhaps this comment should be adjusted?
* For rebuilds, the parent vdev type must support reconstruction using only space maps.
* The only allowable parents are the root vdev or a mirror vdev.

Reply:
Good catch.  I thought I got all of these in an earlier pass, but clearly I missed one.